### ISSUE TITLE
My Images: <port>/<repo> is shifted down on a tile

### ISSUE THREAD

**Issue by vfedorin** [docker/kitematic#2501]
![screen shot 2017-03-24 at 1 24 17 pm](https://cloud.githubusercontent.com/assets/14997994/24290225/3881ec5c-1095-11e7-9e3b-1bad7e29cf1a.png)
### Expected behavior
<port>/<repo> inside a tile.
### Actual behavior
<port>/<repo> is shifted down
### Steps to reproduce the behavior

  1. Go to My Images

**Comment by FrenchBen** [comment]
@vfedorin what's the name of the images that are being displayed?
Can you do a `docker image ls` and post the output here?

**Comment by bla-kw** [comment]
I have the same problem. It happens when you have a private repository with a port number.
For example I have a repository at the server _dockerhub.dev:5001_ the image name is **dockerhub.dev:5001/jenkins/oxid:5.6** and I download it with _docker pull dockerhub.dev:5001/jenkins/oxid:5.6_

```
PS C:\Program Files\Docker\Docker> docker image ls
REPOSITORY                        TAG                 IMAGE ID            CREATED             SIZE
dockerhub.dev:5001/jenkins/oxid   5.6                 68f5b807a468        3 days ago          392 MB
```
In Kitematic the tag is shown as "5001/jenkins/oxid". It takes the server port and strips the real tag.

**Comment by FrenchBen** [comment]
Yup - the private repository parsing of images is on the _todo_ list and is exactly what's causing the above issue.
